tests/cpp/operator/mkldnn.cc

```diff
@@ -93,40 +93,26 @@ TEST(MKLDNN_UTIL_FUNC, MemFormat) {
 
 // Init arrays with the default layout.
 static void InitDefaultArray(NDArray *arr, bool is_rand = false) {
```

Comment:
why do you decide to remove this Init function?

Reply:
No reason to have two inits. We can just have one that covers negative and positives.